MobileCenter/MobileCenter.xcodeproj/xcshareddata/xcschemes/MobileCenter.xcscheme

```diff
@@ -40,7 +40,7 @@
             </BuildableReference>
          </TestableReference>
          <TestableReference
-            skipped = "NO">
```

Comment:
I didn't edit this file directly; I used Xcode. It seems to have added `skipped = "YES"`